<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::EmmaClientWidget</class>
 <widget class="QWidget" name="Event::services::EmmaClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emma Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Export dir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edExportDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="btChooseExportDir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="edExportInterval">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> sec</string>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="singleStep">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Export interval</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>IOF XML 3.0</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chExportXML30">
        <property name="text">
         <string>Enable export IOF XML 3.0 file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btExportXML30">
          <property name="text">
           <string>Export IOF XML 3.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>RACOM</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File name prefix</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edFileName"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chExportStart">
        <property name="text">
         <string>Export start list</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chExportFinish">
        <property name="text">
         <string>Export results</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btExportStart">
          <property name="text">
           <string>Export start list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExportFinish">
          <property name="text">
           <string>Export results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExportSplits">
          <property name="text">
           <string>Export radio codes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edExportDir</tabstop>
  <tabstop>btChooseExportDir</tabstop>
  <tabstop>edExportInterval</tabstop>
  <tabstop>chExportXML30</tabstop>
  <tabstop>btExportXML30</tabstop>
  <tabstop>edFileName</tabstop>
  <tabstop>chExportStart</tabstop>
  <tabstop>chExportFinish</tabstop>
  <tabstop>btExportStart</tabstop>
  <tabstop>btExportFinish</tabstop>
  <tabstop>btExportSplits</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
